### DISCUSSION TITLE
Idea: passing a list of visible WiFi/Wi-Fi APs to games like treasure world?

### DISCUSSION THREAD

**Discussion by FlashlightET** [rafaelvcaetano/melonDS-android#1349] [Ideas]
there is this very obscure game known as "treasure world" that works by seeing visible wifi access points and collecting those as "stars" to fuel a rocket. i havent found any emulators that do this, but it's only fitting to ask about it on an android emulator(?) since that's the most obvious platform for this type of game. is this possible? considerable? if not its fine i could fork it and do it(?)

i do know android apps can do this, because there is an app called Coverage that maps out wifi APs and cell signals and shows their SSIDs pop up on screen. it may even work better than treasure world... or perhaps *too* well due to just how many wifi signals exist these days

**Comment by rafaelvcaetano** [comment]
I had no idea such a game existed! Sounds like a cool idea. However, I need to consider this carefully. Accessing the list of Wi-Fi APs requires the user to give the app the location permission, which is a bit invasive. Specially when it's used for a single game. I will try to think of which approach would be the most suitable here.
If you want to give this a shot, feel free to do so! It's definitely doable.
